### ISSUE TITLE
default admin keeps reappearing

### ISSUE THREAD

**Issue by veonhoon** [DeviceFarmer/stf#575]
hi i made a new admin account and using it to remove the default admin@fakedomain user 

but even after deleting it, when i try the credentials to log in, it works and the user is created again. 

how do i fix this?

**Comment by denis99999** [comment]
@veonhoon ,

Yes, STF has several authentication method (e.g. `mock`, `LDAP`), look at `DEPLOYMENT.md` documentation about that! The default authentication method is the mock one in which anyone can login providing a name and an email.

The admin account is special because it is a built-in object created at database creation time, so if you want to change the admin user your have to re-create the rethindb database and to follow the instructions described at `Running` section of `README.md` documentation.

Also to know more about the administrator level, I advise you to read the `GroupFeature.pdf` documentation, in particular the `10 Database migration` section.
